velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -384,6 +406,206 @@ TEST_F(Re2FunctionsTest, likePattern) {
   EXPECT_EQ(like("abc", "MEDIUM POLISHED%"), false);
 }
 
+TEST_F(Re2FunctionsTest, likeDeterminePatternKind) {
+  EXPECT_EQ(
+      determinePatternKind("_"), std::make_pair(PatternKind::kWildcardOne, 1));
```

Comment:
Consider dropping std::make_pair for readability.

```
{PatternKind::kWildcardOne, 1}
```

Also, consider shortening the test by adding helper function:

```
testPattern("_", PatternKind::kWildcardOne, 1);
```

Reply:
Thanks for the suggestion, shortened the tests using a helper function, std::make_pair is needed here though; the shortened test improves readability considerably.